Gemfile.lock

```diff
@@ -559,7 +551,6 @@ DEPENDENCIES
   therubyracer (~> 0.12)
   timecop
   toastr-rails
-  tzinfo-data (~> 1.2)
```

Comment:
Not sure why this one popped up.  Appeared after running initial install of gems.

Reply:
We had a PR that conditionally included the Gem depending on OS. May need to approach that one differently. 

Good to see another case of it pop up, thanks! It can be removed from the PR or left in -- it'll be addressed either way. Thanks!